### PR TITLE
fix: include `BASE_URL` in image file serialization

### PR DESCRIPTION
**Pull request by BeritJanssen**
This branch fixes a problem on the servers, in which image requests are incorrectly prefixed with `API_ROOT`. Instead, this branch prefixes the `BASE_URL` for the backend image serialization.

### CODE REVIEW

**Review by drikusroor — APPROVED** [review]
Nice!

File: backend/aml/base_settings.py [hunk 1]
```diff
@@ -195,4 +195,4 @@
     }
 }
 
-SUBPATH = os.getenv('AML_SUBPATH', None)
+SUBPATH = os.getenv('AML_SUBPATH', '')
```
Comment (drikusroor) [inline backend/aml/base_settings.py:198]: Now that you are using `BASE_URL` and not `AML_SUBPATH` to prefix the image urls, is this change still needed?

File: backend/experiment/tests/test_views.py [hunk 2]
```diff
@@ -190,7 +191,7 @@ def test_serialize_experiment(self):
         )
         self.assertEqual(
             serialized_experiment['image'], {
-                'file': '/upload/test-image.jpg', 'href': '', 'alt': ''}
+                'file': f'{settings.BASE_URL}/upload/test-image.jpg', 'href': '', 'alt': ''}
```
Comment (drikusroor) [inline backend/experiment/tests/test_views.py:194]: You might as well test the serialization of the `href` & `alt` properties here :)

**Review by drikusroor — APPROVED** [review]
Looks good to me. Did `AML_SUBPATH` not work well in the end?

File: backend/aml/base_settings.py (hunk 1 above)
Comment (BeritJanssen) [reply backend/aml/base_settings.py:198]: No, but I like the empty string fallback better, anyway. I twill still evaluate to `False` in the backend.
